quay-enterprise/quay-tectonic-identity.md

```diff
@@ -0,0 +1,79 @@
+# Quay Enterprise OIDC Auth with Tectonic Identity 
+
+This document explains how to configure Quay Enterprise to use Tectonic Identity as an OIDC provider for authorization and authentication.
+
+## Export certificate information for TLS verification of Tectonic Identity: 
```

Comment:
remove `:`

(if these are headers, they shouldn't have colons at the end of them.)